prow/config.yaml

```diff
@@ -357,6 +357,49 @@ presubmits:
     context: pull-kubernetes-e2e-gce-gpu
     rerun_command: "/test pull-kubernetes-e2e-gce-gpu"
     trigger: "(?m)^/test( all| pull-kubernetes-e2e-gce-gpu),?(\\s+|$)"
+  - name: pull-kubernetes-e2e-gce-gpu-prow
```

Comment:
I think we want to set max_concurrency on this job, too, since it's using a single project.

Reply:
this job is manually triggered so it should be fine. I only intend to trigger it on the testing PR. If all goes well I will follow up with `branches: -master` defaulting to the prow variant.